.github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml

```diff
@@ -0,0 +1,99 @@
+        2. Change the **Title** to include target individual
+        3. Add the label used by the target individual's team (eg: `BAH-526`)
+        4. Do not remove any of the existing labels.
+        5. Enter required fields and select environments you need access to.
+        6. If Production access is required, please use the respective section to provide proof of E-QIP transmittal. If you have not started the E-QIP process, do not submit this ticket. Ask your Vendor Onboarding Representative to confirm.
+        7. Fill in "Additional Notes" with any information requested by the other steps.
+  - type: input
+    id: requestor-name
+    attributes:
+      label: Your Name
+      description: Please provide the name of the target individual
+      placeholder: Jane Doe
+    validations:
+      required: true
+  - type: input
+    id: email
+    attributes:
+      label: Your GitHub handle (Sidekiq Admin UI access is managed through GitHub)
+      description: Please provide the work GitHub handle of the target individual
+      placeholder: "@username"
+    validations:
+      required: true
+  - type: input
+    id: aws-username
+    attributes:
+      label: Your dsvagovcloud AWS user name (if applicable)
+      description: If you have previously been granted AWS access, list your dsvagovcloud AWS user name
+      placeholder: First.Last
+  - type: input
+    id: team-name
+    attributes:
+      label: Team, Role, and Company of the target individual
+      description: Please provide the name of the team, their role on that team, and the name of the company the target individual works for
+      placeholder: Team Moose, Antler Development, Alces LLC
+    validations:
+      required: true
+  - type: dropdown
+    id: cor-name
+    attributes:
+      label: COR Name
+      description: The name of the Contracting Officer's Representative (COR) covering the target individual.
+      options:
+        - Cecila Lee
+        - Courtney Bethea
+        - Crystal Moultrie
+        - Doris Lin
+        - Vilay Senthep
+        - Eunice Garcia
+        - Delano McVay
+        - Jennifer O'Day
+        - Laurene "Reney" Cook
+        - Rolanda Copeland
+        - Other - please specify in 'Additional Notes'
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Please confirm if you are on the VFS Team Roster or Platform Team Roster
+      description: If you are on neither, close this request and reach out to your Product Manager to be added
+      options:
+        - label: "[VFS Team Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)"
+        - label: "[Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)"
```

Comment:
You don't want quotes on either of these `label`s.  I think that's why it's making the markdown look funky maybe? https://github.com/department-of-veterans-affairs/va.gov-team/blob/67bb24ff0c87d91ecc4070192865ed8f7cc147cf/.github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml

Reply:
You _might_ get people saying that they can't view that Platform roster. It's confluence so VFS teams can't see it.

Reply:
I **think** the quotes are fine. The preview is wonky, and it threw me off too, but that's how it is in our other forms.

But the Roster thing is a bummer. I gotta rethink this now, as that kinda screws up my plan here 😢

Reply:
I think we're good here 👍

Reply:
You're right! My bad! I just confirmed by looking at the AWS access template (preview markdown vs real open issue). Looks wonky by renders fine ✅  
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/10993987/3244e5cf-7bde-4bd7-be7e-c9f1e8384df8)